docker-compose.yml

```diff
@@ -6,27 +95,21 @@ services:
     restart: always
     volumes:
       - "/var/run/docker.sock:/tmp/docker.sock:ro"
-      - "./nginx/vhost.d:/etc/nginx/vhost.d"
```

Comment:
I added this for a specific reason, I think it was to be able to overwrite defaults for specific hosts like max payload size

Reply:
Custom nginx config is defined [here](https://github.com/FlowFuse/docker-compose/pull/160/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R67) and attached to the container as a [config](https://github.com/FlowFuse/docker-compose/pull/160/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R101).
During testing the new compose file I couldn't find any reason of mounting this directory (which is empty by default) to the nginx container. Without this binding whole setup worked without issues. That is why I decided to remove this volume binding.